src/test/java/com/google/sps/MealServletTest.java

```diff
@@ -183,6 +183,30 @@ public void getInvalidPathInfoTest() throws IOException, ServletException {
         assertEquals(expected, actual);
     }
 
+    // Get a similar meal from Datastore
+    // Expected result: String in JSON format with id of second meal.
+    
+    @Test
+    public void getIdOfSimilarTest() throws IOException, ServletException {
+        DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+        ds.put(createMealEntity(MEAL_1));
```

Comment:
Add a short comment somewhere (not necessarily here) why meal no. 1 can't be the result (i.e. the "get similar" functionality shouldn't return the same meal as the one the recommendation is based on).